features/step_definitions/series_steps.rb

```diff
@@ -2,6 +2,12 @@
   visit series_url(Series.find_by(title: series))
 end
 
+Given "there are {int} works per page" do |amount|
+  stub_const("ArchiveConfig", OpenStruct.new(ArchiveConfig))
+  ArchiveConfig.ITEMS_PER_PAGE = amount.to_i
+  allow(Work).to receive(:per_page).and_return(amount)
+end
```

Comment:
Hmm. My first thought was I gave you a bad example and we should just be able to do it like blocks per page:
https://github.com/otwcode/otwarchive/blob/34bfb3c642a1b304412315df837050e397b9f97c/features/step_definitions/block_steps.rb#L7-L9

But that didn't work, either. 

What _did_ work was overriding the value of the config with `allow(ArchiveConfig).to receive(:ITEMS_PER_PAGE).and_return(amount)`  _if_ I also explicitly set `per_page: ArchiveConfig.ITEMS_PER_PAGE` in the controller.

But that's gross; we don't want to add extra code just to make our tests pass. 

So I'm thinking if no one swings by this weekend to point out an obvious thing we're both missing, we can probably leave the tests off. While they'd be nice to have, there are lots of places we don't test the pagination -- we can live without them.

Reply:
Bilka2 helped solve the issue! needed to allow `WillPaginate` to receive `:per_page` instead.

Looks like the test is passing now. How's it all looking now?

Reply:
Thanks, @Bilka2! 

Let's make one more tweak to this step and call it `there are {int} works per series page` -- that should remove the temptation to try using it on _other_ pages containing works, where it may or may not work due to the Array vs ActiveRecord difference. Then I think this will be ready for the green checkmark 🎉